src/mscorlib/src/System/Globalization/CompareInfo.cs

```diff
@@ -76,6 +76,8 @@ public partial class CompareInfo : IDeserializationCallback
         [NonSerialized]
         private readonly bool _invariantMode = GlobalizationMode.Invariant;
 
+        private int culture; // Do not rename (binary serialization)
```

Comment:
please add a comment that this field is there only to make the full framework happy with the serialization